app/frontend/src/components/form/OrgBookSearch.vue

```diff
@@ -65,7 +80,17 @@ export default {
         // For this use, want to emit just the text
         typeof value === 'object' && value !== null ? value.text : value
       );
+
+      // show Org Book warning and help message when user enters a business name not found in the Org Book
+      this.showOrgBookHelp = (value && !this.foundInOrgBook(value.text));
     },
+    // Check if user input matched an Org Book suggestion
+    foundInOrgBook(inputValue){
+      const arrayOfOrgBookValues = this.entries.map( result => {
+        return result.names[0].text;
+      });
+      return arrayOfOrgBookValues.includes(inputValue);
```

Comment:
This can be simplified into a one liner:
```js
return this.entries.map(result => result.names[0].text).includes(inputValue);
```

Reply:
One liners aren't always better if the next developer will need to spend extra time figuring out how it works. especially on this app which we are hopefully going to be handing over to someone else to maintain.    However if Tim likes your suggested change and it doesn't make it harder to maintain, then by all means make it into a one-liner.   :)